clients/python/test/e2e/test_solvers_api.py

```diff
@@ -53,6 +38,9 @@ def test_get_jobs(configuration: osparc.Configuration):
         tmp_iter = solvers_api.get_jobs(
```

Comment:
MIONR: is `solvers_api.get_jobs` called like that? Listing jobs should be called `solvers_api.list_jobs`. All listings are by convention called that way. 

MINOR: I find interesting the naming `tmp`. A generation is indeed something of one use but the idea is that the use does not even feel it, i.e. the user would write something like

```python

for job in solvers_api.list_jobs(...):
    print(job)
    # do something with job
```

Reply:
OK, I can call it `list_jobs` if you want. But for me "list" is a loaded term in Python. I would expect such a method to return me a list (i.e. it seems to me to violate the principle of lease surprise :wink:). 

```python

for job in solvers_api.list_jobs(...):
    print(job)
    # do something with job
```
works now. However more than that one can also do `len(solvers_api.list_jobs(...))` to get the number of elements the iterator can produce. I find that quite useful. That's why I used te `tmp_...`-thing in the test. To get an object I could call `len` on.